legal-api/src/legal_api/models/business.py

```diff
@@ -283,6 +285,27 @@ class AssociationTypes(Enum):
     batch_processing = db.relationship('BatchProcessing', lazy='dynamic')
     jurisdictions = db.relationship('Jurisdiction', lazy='dynamic')
 
+    @dataclass
+    class AffiliationSearchDetails:  # pylint: disable=too-many-instance-attributes
```

Comment:
Move this data class to search_service. We are moving all models to https://github.com/bcgov/lear/tree/main/python/common/business-registry-model